app/blueprints/self_serve/routes.py

```diff
@@ -147,22 +136,42 @@ def build_section():
                 "hover_info": {"title": f["builder_display_name"], "pages": f["pages"]},
             }
         )
-    return render_template("build_section.html", available_forms=available_forms, form=form)
-
+        # save to db here
```

Comment:
shall we add TODO's for these then we can easily identify these and if possible add Jira tickets if needed

Reply:
done :)